cmd/lakefs/cmd/setup.go

```diff
@@ -100,15 +106,17 @@ var setupCmd = &cobra.Command{
 	},
 }
 
-func setupLakeFS(ctx context.Context, cfg *config.BaseConfig, metadataManager auth.MetadataManager, authService auth.Service, userName string, accessKeyID string, secretAccessKey string) (*model.Credential, error) {
-	initialized, err := metadataManager.IsInitialized(ctx)
-	if err != nil || initialized {
-		// return on error or if already initialized
-		return nil, err
+func setupLakeFS(ctx context.Context, cfg *config.BaseConfig, metadataManager auth.MetadataManager, authService auth.Service, userName string, accessKeyID string, secretAccessKey string, noSetupCheck bool) (*model.Credential, error) {
```

Comment:
Are we OK with this [event](https://github.com/treeverse/lakeFS/blob/master/cmd/lakefs/cmd/setup.go#L94) being sent 2+ times (each setup --no-check request)?

Reply:
- Good point - I'll check if there will be any BI issue.
- As I saw it the user started with the fresh setup so the metric is in place. Also the alternative would probably setup a new lakeFS and running setup there (for fresh enterprise).
- Other enterprise customers will be running setup as part of setup a new instance.

Reply:
Thanks @nopcoder @Isan-Rivkin 
No issue sending it more than once.
Setup event can be sent more than once regardless of the conversion to enterprise. How can we mark the timestamp of this installation's transition from OSS to enterprise according to events being sent? Can we distinguish between OSS and enterprise based on the events we are reporting as of today?